lib/dry/configurable/class_methods.rb

```diff
@@ -11,8 +11,16 @@ module ClassMethods
       def inherited(subclass)
         super
 
-        subclass.instance_variable_set("@_settings", _settings.dup)
-        subclass.instance_variable_set("@_config", config.dup) if respond_to?(:config)
+        new_settings = _settings.dup
+
+        subclass.instance_variable_set(:"@_settings", new_settings)
+
+        # Only classes **extending** Dry::Configurable have class-level config. When
+        # Dry::Configurable is **included**, the class-level config method is undefined because it
+        # resides at the instance-level instead (see `Configurable.included`).
```

Comment:
I left this note here because I remember being confused in the past about why we may not respond to `config` despite it being defined right here in this module. Hopefully this helps alleviate any confusion from readers in the future.